storage-engine/titan-configuration.md

```diff
@@ -146,6 +162,10 @@ To fully disable Titan for all existing and future data, you can follow these st
     enabled = false
     ```
 
+### Data convertion speed from Titan to RocksDB
+
+Because the values in Titan Blob files are not contiguous, and Titan's cache is at the value level, the Blob Cache does not help during compaction. The speed from Titan to RocksDB is an order of magnitude slower than the speed from RocksDB to Titan. In the test, it takes 12 hours for a volume of 800 GiB Titan data on a TiKV node to be converted to RocksDB by tikv-ctl to do full compaction.
```

Comment:
```suggestion
Because the values in Titan Blob files are not contiguous, and Titan's cache is at the value level, the Blob Cache does not help during compaction. The speed from Titan to RocksDB is an order of magnitude slower than the speed from RocksDB to Titan. In the test, it takes 12 hours to convert a volume of 800 GiB Titan data on a TiKV node to RocksDB by tikv-ctl in a full compaction.
```